### ISSUE TITLE
the ability to specify `--unknown-slide`

### ISSUE THREAD

**Issue by aguang** [nf-core/spatialvi#99]
### Description of feature

It doesn't seem like with the nextflow pipeline that there is a way to specify `--unknown-slide` (as described here: https://www.10xgenomics.com/support/software/space-ranger/latest/analysis/inputs/image-slide-parameters) as an option instead of `--slide` and `--area`. If there is a way to do so, could you point me to it? And if not, can this be implemented as a feature given that it's a space ranger option?

**Comment by fasterius** [comment]
You're right, this is currently not included in the pipeline, although it does seem like this is more something that should get included in the `spaceranger/count` module itself rather than the `spatialvi` pipeline. I will see what I can do regarding this.

**Comment by fasterius** [comment]
A PR to the `spaceranger/count` module is up (see link above), but will likely not be merged until after the holidays are over (see a Slack thread here: https://nfcore.slack.com/archives/CJRH30T6V/p1734510282197769). In the meantime, you can always change the module code yourself according to the PR, if you're keen to start using the functionality, @aguang.
